HIP/hip-412.md

```diff
@@ -143,18 +163,16 @@ Including the mime type allows applications to properly handle the file and grea
 
 An NFT creator has the option of using either "metadata" or "metadata_uri". Metadata should be considered the default behaviour as it minimizes the number of calls that need to be made. Metadata_uri should be used in specific situations where defining the metadata object within the base metadata file is inadequate.
 
-The opensea format defines "files" under the "properties" field. This standard places files in the root of the metadata in order to accommodate non-image NFT’s. It is recommended for NFT’s with the "opensea" format that they follow the opensea standard of placing files under the properties field, for cross-compatibility with other NFT’s, and not define a "files" field in the metadata root. The idea comes from the Solana NFT Standard from Metaplex: https://docs.metaplex.com/token-metadata/specification
-
 #### format
 
 For the optional fields of attributes and properties, as well as any additional fields above the required fields described in this schema, "format" defines the specific schema which is used by this NFT.
 This allows NFT creators, communities and platforms to explicitly define the schema that they are using, which simplifies implementation for other projects hoping to use the same definition.
 
 To reduce errors, "format" should be lower-case. For robustness, galleries and viewers should interpret format in a case-insensitive way to account for mistakes.
 
-If format is not defined, it is assumed to be "opensea", which is described here: https://docs.opensea.io/docs/metadata-standards . "opensea" can (and should) also be explicitly defined by format, where applicable, rather than leaving it blank. 
+For the purposes of this base metadata schema we also define `"format": "none"`, which is a catch-all that explicitly states that the NFT only follows the base fields of the schema but may define any number of additional fields.
```

Comment:
Good idea! I do like "none" because I want to force users to use the "format" field. Then we can also keep the "none" reference implementation examples in the document which are still valuable? We can indeed use this to further list `HIP412@1.0.1` once we make an update so we have a backlog of all standards.

Reply:
Is just not including format the same as `format: none`? What is the purpose of `none`? I see no reason to have it at all. I may be missing something here.

I see it simply as, if you use this standard then set `format` to "HIP412@1.0.0" to let parsers know this is the structure you followed.

If they're not following the HIP 412 standard then there's no reason for them to add format.

Reply:
Removed it!